sparkly/utils.py

```diff
@@ -93,6 +181,45 @@ def kafka_get_topics_offsets(host, topic, port=9092):
     return offsets
 
 
+class lru_cache(object):
```

Comment:
Got it, thanks. honestly I'd prefer to drop LRU part of this. It's sounds like "too much" for the feature